Cargo.toml

```diff
@@ -88,7 +88,7 @@ ra-ap-rustc_lexer = { version = "0.40.0", default-features = false }
 ra-ap-rustc_parse_format = { version = "0.40.0", default-features = false }
 ra-ap-rustc_index = { version = "0.40.0", default-features = false }
 ra-ap-rustc_abi = { version = "0.40.0", default-features = false }
-ra-ap-rustc_pattern_analysis = { version = "0.40.0", default-features = false }
+ra-ap-rustc_pattern_analysis = { version = "0.42.0", default-features = false }
```

Comment:
Can we bump the other ra-ap-rustc crates here as well? Otherwise we get duped dependencies of those and usually they don't contain any abi breakage

Reply:
Done, there wa sonly a tiny API change